specs-rand-patch.md

```
@@ -0,0 +1,63 @@
+
+### AC2: Test suite passes
+After applying the patch, `cargo test` must pass without rand-related panics or failures. This includes running property-based tests in all affected crates:
+- diffguard-testkit (strategies)
+- diffguard-types (properties)
+- diffguard (baseline_mode_properties)
+- diffguard-domain (properties)
+- diffguard-core (properties, property_test_checkstyle, property_tests_escape_xml)
+- diffguard-diff (properties)
+- bench (property_tests)
+
+### AC3: Only Cargo.toml is modified
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)